### PR TITLE
Update dictionary.js

### PR DESCRIPTION
**Pull request by Bee-Stack**

### CODE REVIEW

File: Vanilla/Resources/dictionary.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// Weather API codes for current weather conditions
```
Comment (Bee-Stack) [inline Vanilla/Resources/dictionary.js:1]: This comment makes it easy for everyone to understand what these keycodes do.

File: Vanilla/Resources/dictionary.js [hunk 2]
```diff
@@ -12,6 +13,7 @@ let present = [{
 	prefix: "it is"
 }];
 
+// Weather API codes for forecast conditions
```
Comment (Bee-Stack) [inline Vanilla/Resources/dictionary.js:16]: Here again, for forecast conditions, it makes a lot of sense to add this comment.

**Review by Bee-Stack — COMMENTED** [review]
Please see comments

File: Vanilla/Resources/dictionary.js (hunk 1 above)
Comment (sachinbhatnagar) [reply Vanilla/Resources/dictionary.js:1]: Ok, will change

**Review by sachinbhatnagar — APPROVED** [review]

**Pull request merged**